include/Core/UI/Page/LogListPage.php

```diff
@@ -206,16 +206,20 @@ public function load_view_logs_assets( $hook ) {
 			return;
 		}
 
+		global $wp_scripts;
 		$email_log      = email_log();
 		$plugin_dir_url = plugin_dir_url( $email_log->get_plugin_file() );
 
 		wp_enqueue_style( 'jquery-ui-css', $plugin_dir_url . 'assets/vendor/jquery-ui/themes/base/jquery-ui.min.css', array(), '1.12.1' );
 		wp_enqueue_style( 'el-view-logs-css', $plugin_dir_url . 'assets/css/admin/view-logs.css', array( 'jquery-ui-css' ), $email_log->get_version() );
 
+		$ui  = $wp_scripts->query( 'jquery-ui-core' );
+		wp_enqueue_style( 'jquery-ui-smoothness', $plugin_dir_url . 'assets/vendor/jquery-ui/themes/smoothness/jquery-ui.min.css', array(), $ui->ver );
```

Comment:
Let's hardcode the version here.

We are not using the ui-core css from WordPress core, but directly from the vendor directory. So it should be the version of the package from npm.

Reply:
> Let's hardcode the version here.

Done.